AdobeStockAsset/Model/AppendAttributes.php

```diff
@@ -7,12 +7,11 @@
 
 namespace Magento\AdobeStockAsset\Model;
 
-use Magento\MediaGalleryApi\Model\Asset\Command\GetByIdInterface;
-use Magento\AdobeStockAsset\Model\ResourceModel\Asset\LoadByIds;
+use Magento\AdobeStockAsset\Model\ResourceModel\Asset\Command\LoadByIds;
 use Magento\Framework\Api\AttributeValueFactory;
-use Magento\Framework\Api\Search\Document;
-use Magento\Framework\Api\Search\SearchResultInterface;
+use Magento\Framework\Api\Search\{Document,SearchResultInterface};
```

Comment:
@coderimus could you provide some documentation about this way of declare use?
Is this what you mentioned during the call?

Thanks

Reply:
Hello @diazwatson ,

[In the PHP 7.0 the Group Namesapce Declaration was added ](https://wiki.php.net/rfc/group_use_declarations) This approach provides the ability to group imported namespaces a make them more structured. In the PSR-12 which approved to be used (as I know) we have some rules about this feature. You can find information here: [3. Declare Statements, Namespace, and Import Statements](https://www.php-fig.org/psr/psr-12/#3-declare-statements-namespace-and-import-statements)

Reply:
Thanks :)